internal/cmd/keys.go

```diff
@@ -114,11 +122,20 @@ func newKeysRemoveCmd() *cobra.Command {
 				return fmt.Errorf("invalid access key response, there should only be one access key that matches a name, but multiple were found")
 			}
 
-			err = client.DeleteAccessKey(keys[0].ID)
+			key := keys[0]
+
+			err = client.DeleteAccessKey(key.ID)
 			if err != nil {
 				return err
 			}
 
+			issuedFor := key.IssuedForName
+			if issuedFor == "" {
+				issuedFor = key.IssuedFor.String()
+			}
+
+			fmt.Fprintf(os.Stderr, "Deleted access key %q issued for %q\n", key.Name, issuedFor)
```

Comment:
This isn't related to optional access key name but remove doesn't currently have any outputs